src/icalendar/parser.py

```diff
@@ -308,7 +332,62 @@ def value(self) -> VALUE | str | None:
             Applications MUST preserve the value data for x-name and iana-
             token values that they don't recognize without attempting to
             interpret or parse the value data.
+
+        For convenience, using this property, the value will be converted to
+        an uppercase string.
+
+        .. code-block:: pycon
+
+            >>> from icalendar import Parameters
+            >>> params = Parameters()
+            >>> params.value = "unknown"
+            >>> params
+            Parameters({'VALUE': 'UNKNOWN'})
+
+        """
+
+    def to_jcal(self, exclude_utc=False) -> dict[str, str]:
+        """Return the jcal representation of the parameters.
+
+        Args:
+            exclude_utc (bool): Exclude the TZID parameter if it is UTC
+        """
+        jcal = {k.lower(): v for k, v in self.items() if k.lower() != "value"}
+        if exclude_utc and jcal.get("tzid") == "UTC":
+            del jcal["tzid"]
+        return jcal
+
+    @single_string_parameter()
+    def tzid(self) -> str | None:
+        """The TZID parameter from :rfc:`5545`."""
+
+    def is_utc(self):
+        """Wether the TZID parameter is UTC."""
+        return self.tzid == "UTC"
+
+    def update_tzid_from(self, dt: datetime | time | Any) -> None:
+        """Update the TZID parameter from a datetime object.
+
+        This sets the TZID parameter or deletes it according to the datetime.
+        """
+        if isinstance(dt, (datetime, time)):
+            self.tzid = tzid_from_dt(dt)
+
+    @classmethod
+    def from_jcal(cls, jcal: dict[str : str | list[str]]):
+        """Parse jcal parameters."""
+        return cls(jcal)
```

Comment:
Validate types and raise `JCalParsingError`

Reply:
Yes, is validated now.